### PR TITLE
Updates to website

### PR DESCRIPTION
**Pull request by tlegen-k**
## Related Issues and Dependencies
Fixed issues with broken links, typos  and styling.
…

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

… Explain your changes.

### CODE REVIEW

File: docs/source/predictors/latest.rst [hunk 1]
```diff
@@ -13,9 +13,8 @@ algorithm but rather approximates the latest software stack resolution.
...
 resolver.
 
-Check `termial random
-<https://medium.com/@fridex/termial-random-for-prioritized-picking-an-item-from-a-list-a65a4f563224>`__
-and `its optimized form
-<https://medium.com/@fridex/optimizing-termial-random-by-removing-binomial-coefficient-e39b9ca7aaa3>`__
-which is used to prioritize picking more recent releases considering version
-specifier.
+..  Check `termial random
+  <https://medium.com/@fridex/termial-random-for-prioritized-picking-an-item-from-a-list-a65a4f563224>`__
```
Comment (fridex) [inline docs/source/predictors/latest.rst:17]: This one was moved to https://dev.to/fridex/termial-random-for-prioritized-picking-an-item-from-a-list-22jh

File: docs/source/architecture.rst [hunk 2]
```diff
@@ -31,7 +31,7 @@ Thoth is deployed from `thoth-station/thoth-application repo
 using using `kustomize <https://kustomize.io/>`__.
 
 See `Requirements for AICoE-CI & Thoth deployment
-<https://docs.google.com/document/d/14itc_SqPrK_rOai8ETVMhP7MBhdG3K8ql0iW8g1P6YE/>`__
+<https://github.com/AICoE/aicoe-ci>`__
```
Comment (fridex) [inline docs/source/architecture.rst:34]: The linked document is internal so it makes sense to link the repo 👍🏻

File: docs/source/predictors/latest.rst [hunk 3]
```diff
@@ -13,9 +13,8 @@ algorithm but rather approximates the latest software stack resolution.
...
+  and `its optimized form
+  <https://medium.com/@fridex/optimizing-termial-random-by-removing-binomial-coefficient-e39b9ca7aaa3>`__
+  which is used to prioritize picking more recent releases considering version specifier.
```
Comment (fridex) [inline docs/source/predictors/latest.rst:20]: I need to re-publish this one. It was moved to a more open platform but sits there as a draft. Feel free to remove it.

**Review by fridex — APPROVED** [review]
Nice, thanks! Just minor comments (can be addressed in another PR if you want). I'm open to merge this. 👍🏻 Thanks!

File: docs/source/predictors/latest.rst (hunk 1 above)
Comment (tlegen-k) [reply docs/source/predictors/latest.rst]: I have update the first link with the web-address provided

File: docs/source/predictors/latest.rst (hunk 3 above)
Comment (tlegen-k) [reply docs/source/predictors/latest.rst:20]: Commented it out for now. Can update this link as well once you publish it.

**Review by tlegen-k — COMMENTED** [review]
Updated according to comments provided

**Comment by fridex** [comment]
> Updated according to comments provided

Thanks! Changes look good to me.

By the way, it might be good to check GitHub contributing guide. I see you are working on master branch of your fork which can make some issues in the future:

https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project

It might be also good to squash commits and provide commit messages that can be further used when we generate release notes.

I think @harshad16 has a demo on how to use git in the upcoming interns bootcamp and how to contribute to a project.

/lgtm

**Comment by tlegen-k** [comment]
> > Updated according to comments provided
> 
> Thanks! Changes look good to me.
> 
> By the way, it might be good to check GitHub contributing guide. I see you are working on master branch of your fork which can make some issues in the future:
> 
> https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project
> 
> It might be also good to squash commits and provide commit messages that can be further used when we generate release notes.
> 
> I think @harshad16 has a demo on how to use git in the upcoming interns bootcamp and how to contribute to a project.
> 
> /lgtm

Thank you for feedback!
I will read the guide and will make more descriptive commit messages from now on.

**Pull request merged**